docs/DailyBuilds.md

```diff
@@ -4,9 +4,10 @@ Daily builds include the latest source code changes. They are not supported for
 
 If you want to download the latest daily build and use it in a project, then you need to:
 
-* Obtain the latest [build of the .NET Core SDK](https://github.com/dotnet/core-sdk#installers-and-binaries).
+* Obtain the latest [build of the .NET Core SDK](https://github.com/dotnet/installer#table).
 * Add a NuGet.Config to your project directory with the following content:
 
+**.NET 7**
```

Comment:
Don't just embolden here or for **.NET 8**
```suggestion
## .NET 7

```

(blank line between headings and following text or lists also recommended)